authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/manager/AuthenticatorBridgeManagerImpl.kt

```diff
@@ -58,7 +60,9 @@ internal class AuthenticatorBridgeManagerImpl(
      */
     private val mutableSharedAccountsStateFlow: MutableStateFlow<AccountSyncState> =
         MutableStateFlow(
-            if (isBitwardenAppInstalled()) {
+            if (isBuildVersionBelow(Build.VERSION_CODES.S)) {
```

Comment:
Can we split this up into a `when` now?

Reply:
Ah, yep, that's fair!

b97e8f12560f9413f57e2415d53f9301338a2d94